rust/hook-janitor/src/webhooks.rs

```diff
@@ -397,36 +520,58 @@ impl WebhookCleaner {
 
         let mut tx = self.start_serializable_txn().await?;
 
-        let (completed_row_count, completed_agg_row_count) = {
-            let completed_row_count = self.get_row_count_for_status(&mut tx, "completed").await?;
-            let completed_agg_rows = if self.hog_mode {
-                // Hog mode doesn't need to send metrics to Kafka (and can't aggregate by
-                // plugin anyway), so we can skip this.
-                vec![]
-            } else {
-                self.get_completed_agg_rows(&mut tx).await?
-            };
-            let agg_row_count = completed_agg_rows.len() as u64;
+        let completed_row_count = self.get_row_count_for_status(&mut tx, "completed").await?;
+        let completed_agg_row_count = if self.hog_mode {
```

Comment:
These giant "if hog_hooks" switches are a little ugly, but I think if we treat the whole existence of that bool as transitory it's alright for now.

Reply:
Yeah, they're ugly but the goal is that they will be removed ASAP.